tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/HttpClient.scala

```diff
@@ -55,11 +56,23 @@ class HttpClient private (baseUrl: Uri, httpExt: HttpExt)(implicit
   )(implicit um: FromEntityUnmarshaller[A]): IO[Assertion] =
     requestAssert(POST, url, Some(body), identity, extraHeaders)(assertResponse)
 
+  def post2[A](url: String, body: IO[Json], identity: Identity, extraHeaders: Seq[HttpHeader] = jsonHeaders)(
```

Comment:
Temporary ?

Reply:
Will be renamed :) but it's sort of needed